### ISSUE TITLE
Reproduction of Evaluation

### ISSUE THREAD

**Issue by stefan-baumann** [hwjiang1510/RayZer#7]
Hi,

Thank you for this excellent work!

I'm currently trying to reproduce your evaluations and would appreciate a few clarifications to enable that. The most helpful artifact would be some file that maps video -> exact frame indices of inputs & outputs for the three datasets in each reported setting. Short of that, the points below would help mirror your evaluation protocol as closely as possible:

**NVS Evaluation**

1. Can you confirm the exact counts for input & target views at eval time for each dataset and whether they are identical to the final stage of the training curriculum?
2. Frame index ranges & selections
   1. How are these ranges partitioned into inputs vs. targets? What precisely do "even" and "random" sampling mean in this context?
   2. When sampling, do you enforce that the canonical view is included as an input view?
3. Are all K = #input + #target views passed to the camera estimator to predict poses?

**Pose Evaluation**

1. Are the views the same as used for the NVS evaluation, combining the input and output views into one set of views for the pose estimator?
2. Can you give more details on the probe training? How exactly was the probe trained and what were its hyperparameters? Were the targets normalized/aligned in any way?
2. Are rotations and translations computed w.r.t. the canonical view? If not, what is the reference?
3. For the translation evaluation, are the predicted poses aligned to COLMAP before error computation? If so, by which method?
4. What exactly do the translation thresholds represent? Are those absolute or relative distances, and in which coordinate system?

Thanks a lot in advance for clarifying these points! Having these details would make reproducing and fairly comparing results much more straightforward.

Best, Stefan

**Comment by szymanowiczs** [comment]
@hwjiang1510 thanks for the excellent work! Similar to @stefan-baumann I'd like to request if you could release the exact input and target views used for evaluation - this would greatly help with reproducing your results.

Many thanks,
Stan

**Comment by hwjiang1510** [comment]
Hi everyone,

I included the core code and the model weights (trained for 52k iter) [here](https://drive.google.com/drive/folders/1GRVorywGVdqExqjn_Y-lE7RMTme1FNKt?usp=sharing) for your reference.

I will clean up the evaluation code recently.

**Comment by szymanowiczs** [comment]
Many thanks for this @hwjiang1510 ! Any chance you could provide a json with view indices you used for evaluating your models in Tables 1. and 2.? I'd love to be able to run other methods on the same split to get comparable results.

**Comment by hwjiang1510** [comment]
Great suggestion!

I have uploaded the detailed evaluation results on DL3DV [here](https://drive.google.com/drive/folders/1V9sS2R61_jTVss2-dSCMlmyDpjpb1Ulc?usp=sharing).

The results include view indices, renderings, comparison w. GT and per-view metrics.
The results are produced by a model with 32 transformer layers trained 100K iterations. This training setting follows the standard LVSM training and is different from the setting used in paper (as we had limited budget then). The weight and config of the new model is also included.

**Comment by szymanowiczs** [comment]
That's awesome thanks a lot @hwjiang1510 ! Just a quick question - how do the indices in the folder map to scene ids? is there a way to recover scene id easily for each folder names like 000{scene_number} ?
